### PR TITLE
Add source-mapping to dash-bio

### PR DESCRIPTION
**Pull request by HammadTheOne**
## About

This PR adds source-mapping to facilitate easier debugging and code exploration after creating chunks/bundles.

## Description of changes

Adds source mapping via plugin to `webpack-config` and updates `init.py` to include `.map.js` files.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,5 +1,6 @@
-include dash_bio/bundle.js
 include dash_bio/dash_bio-shared.js
+include dash_bio/dash_bio-shared.js.map
+include dash_bio/bundle.js
```
Comment (alexcjohnson) [inline MANIFEST.in:3]: also `bundle.js.map`?

File: webpack.config.js [hunk 2]
```diff
@@ -62,7 +63,11 @@ module.exports = {
         }
     },
     plugins: [
-        new WebpackDashDynamicImport()
+        new WebpackDashDynamicImport(),
+        new webpack.SourceMapDevToolPlugin({
+            filename: '[file].map',
+            exclude: ['async-plotlyjs']
```
Comment (alexcjohnson) [inline webpack.config.js:69]: dash-bio doesn't have an `async-plotlyjs` chunk so I don't think this is doing anything... `NeedlePlot` and `AlignmentChart` both use `react-plotly.js`, looks like plotly.js is included in `async-alignment` and needleplot pulls it from there.

As long as it doesn't take too long to build the sourcemaps though, we can keep them all.

File: MANIFEST.in (hunk 1 above)
Comment (HammadTheOne) [reply MANIFEST.in:3]: Thanks for catching that, added in c9eef3c.

Comment (alexcjohnson) [reply MANIFEST.in:3]: ?? I don't see a change to `MANIFEST.in` in that commit.

Comment (HammadTheOne) [reply MANIFEST.in:3]: Sorry about that, I was working on 2 concurrent branches and added my changes to the other one by mistake. Fixed in 
331217b .

File: webpack.config.js (hunk 2 above)
Comment (HammadTheOne) [reply webpack.config.js]: Removed in 59586a1.

**Review by alexcjohnson — APPROVED** [review]
💃 Looks great, works perfectly 🎉

**Pull request merged**